chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/DeveloperManagerTest.java

```diff
@@ -106,7 +101,7 @@ public static void setUpClass() throws Exception {
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
-        Mockito.when(ff4j.check(FeatureList.EFFECTIVE_RULE_DATE_PLUS_ONE_WEEK)).thenReturn(false);
+        Mockito.doReturn(false).when(ff4j).check(FeatureList.EFFECTIVE_RULE_DATE);
```

Comment:
This was ERD+1Week, but is now ERD. Is that on purpose?

Reply:
I... don't know.  None of the managers being tested in this class reference FF4J at all. I can remove it from the test class entirely and everything still passes. I'll check that in.